### PR TITLE
[WIP] Refactor meetings controller2

### PR DESCRIPTION
**Pull request by hmasila**
# Description 

## More Details

It's all about cleaning that code

image

## Corresponding Issue
https://github.com/ifmeorg/ifme/issues/840

# Screenshots

# Test Coverage

✅ 
🚫

### CODE REVIEW

**Comment by jm96441n** [comment]
Can the base for this be set to the other refactor branch so the changes are a bit more visible?

File: app/controllers/meetings_controller.rb [hunk 1]
```diff
@@ -45,58 +26,40 @@ def comment
 
   # rubocop:disable MethodLength
   def delete_comment
```
[the remark this answers is not in the document]
Comment (hmasila) [reply app/controllers/meetings_controller.rb]: Yes, I agree with you that this method seems out of place. I am not very familiar with the codebase that is why I am "refactoring in place" - leaving everything as it was. Maybe with more context I will be able to implement better practices on functionality

**Comment by julianguyen** [comment]
Omg your cleaning stock images lol

File: app/controllers/meetings_controller.rb [hunk 2]
```diff
@@ -83,7 +82,8 @@ def create
   def update
     if @meeting.update(meeting_params)
       @meeting_members = @meeting.members
-      notify_members(@meeting, @meeting_members, 'update_meeting')
+      send_notification(@meeting, @meeting_members, 'update_meeting')
+
 
       respond_to do |format|
         format.html { redirect_to meeting_path(@meeting.slug) }
```
Comment (hmasila) [inline app/controllers/meetings_controller.rb]: @julianguyen Is this the expected response `render json: @meeting.errors, status: :unprocessable_entity` in https://github.com/ifmeorg/ifme/pull/1080/files#diff-ec628dfcdda37a04a922f83384b41725L91

Comment (julianguyen) [reply app/controllers/meetings_controller.rb]: Hmmm good question 🤔  I don't think so :/ 
I think you can actually get rid of that respond_to block and replace it with `redirect_to_path(meeting_path(@meeting.slug))`

Comment (hmasila) [reply app/controllers/meetings_controller.rb]: Does this apply to the other `respond to` blocks or this one specifically? @julianguyen

Comment (julianguyen) [reply app/controllers/meetings_controller.rb]: Yeah, I would say so excluding the ones with alerts or notices.

Comment (julianguyen) [reply app/controllers/meetings_controller.rb]: I think you can get rid of the respond block and format.json for redirectrs with alerts or notices and just do: `redirect_to(group_path(group_id), alert: t('meetings.leave.error'))` for instance.

**Comment by hmasila** [comment]
@julianguyen I'm not sure I got your comment https://github.com/ifmeorg/ifme/pull/1080#discussion_r222834125 correctly. Please review the recent commit and advice accordingly

**Comment by hmasila** [comment]
I will just add a few more tests to get the coverage to 50% then merge it

**Comment by hmasila** [comment]
Actually, I will add the tests on a separate PR because the untested methods are quite a number

**Pull request merged**